src/com/sap/piper/Utils.groovy

```diff
@@ -68,9 +68,7 @@ def unstashStageFiles(Script script, String stageName, List stashContent = []) {
     stashContent += script.commonPipelineEnvironment.configuration.stageStashes?.get(stageName)?.unstash ?: []
 
     script.deleteDir()
-    unstashAll(stashContent)
-
-    return stashContent
+    return unstashAll(stashContent)
```

Comment:
See comments [here](https://github.com/SAP/jenkins-library/pull/4377/files#r1206309496).
And should we really have separate PRs for this or does it belong together?

Reply:
Copied over [this comment](https://github.com/SAP/jenkins-library/pull/4377/files#r1206315507):

> I wonder if `unstashedContent += unstash(stashContent[i])` in [line 111](https://github.com/SAP/jenkins-library/pull/4377/files#diff-f8351446966fe1c7d84083a9afbc0a6bace4ab0191fb4998bd6a8fffd506386eR111)  works. Jenkins `unstash` returns null in my tests.